src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -45,16 +45,16 @@ Refer to our
 [documentation and examples](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app)
 for monitoring service entities using OpenTelemetry.
 
-#### Required attributes
+#### Required attributes [#service-required-attributes]
 
 * [`service.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/service): The service name you see in the UI comes from this value.
 
-#### Recommended attributes [#recommended-attributes]
+#### Recommended attributes [#service-recommended-attributes]
```

Comment:
@alanwest need to differentiate between the required / recommended / entity tags sections for each entity type.